content/W2015/W15-event-uwarcmovienight.md

```diff
@@ -0,0 +1,20 @@
+Title: WiCS and UWARC Movie Night
+Date: 2015-02-12 19:00
+Category: Events
+Tags: social
+Slug: uwarc-movie-night
+Author: Melissa Tedesco
+Summary: Join us for a post-midterm movie night at POETs.  
```

Comment:
POETS is with a capital S: http://engsoc.uwaterloo.ca/services/poets

If you can fix these three typos that would be great!